packages/hoppscotch-common/src/components/app/PaneLayout.vue

```diff
@@ -30,25 +30,40 @@
       </Splitpanes>
     </Pane>
     <Pane
-      v-if="SIDEBAR && hasSidebar"
+      v-if="mdAndLarger && SIDEBAR && hasSidebar"
       :size="PANE_SIDEBAR_SIZE"
       min-size="20"
       class="flex flex-col !overflow-auto bg-primaryContrast"
     >
       <slot name="sidebar" />
     </Pane>
+
+    <!-- Mobile Collections Layout -->
+    <SmartSlideOver
+      v-if="!mdAndLarger && SIDEBAR && hasSidebar"
+      :show="show"
+      :title="t('tab.collections')"
```

Comment:
Pass the title as a prop. If we want to use SlideOver later from a different page/component we can do that.

Reply:
Title is being passed as a prop to SlideOver Component.